canvas_modules/harness/cypress/support/properties/properties-cmds.js

```diff
@@ -53,3 +34,43 @@ Cypress.Commands.add("openSubPanel", (title) => {
 	cy.get(".properties-icon-button-label").contains(title)
 		.click();
 });
+
+Cypress.Commands.add("clickAction", (actionName) => {
+	cy.get(".properties-action-button button").contains(actionName)
+		.click();
+});
+
+Cypress.Commands.add("clickSubPanelButtonInRow", (controlId, row) => {
+	cy.get("div[data-id='properties-" + controlId + "']").find(".properties-subpanel-button")
+		.then((idx) => {
+			idx[row].click();
+		});
+});
+
+Cypress.Commands.add("setRenameSubPanelLabel", (controlId, labelText) => {
+	cy.get("div[data-id='properties-" + controlId + "']").find("input")
+		.type(labelText);
+});
+
+Cypress.Commands.add("getWideFlyoutPanel", (panelName) => {
+	cy.get(".properties-wf-content.show")
+		.then((wideFlyoutPanels) => {
+			let panel = null;
+			for (var idx = 0; idx < wideFlyoutPanels.length; idx++) {
+				const flyout = wideFlyoutPanels[idx];
+				if (flyout.textContent.includes(panelName)) {
+					panel = flyout;
+					break;
+				}
+			}
+			return panel;
+		});
+});
+
+Cypress.Commands.add("saveWideFlyoutHavingName", (panelName) => {
```

Comment:
I would rename this saveWideFlyout
The name part is in the parameter so we don't need it in the method name as well.